mmv1/third_party/terraform/resources/resource_compute_instance_group_manager.go.erb

```diff
@@ -317,9 +317,58 @@ func resourceComputeInstanceGroupManager() *schema.Resource {
 				Description: `When used with wait_for_instances specifies the status to wait for. When STABLE is specified this resource will wait until the instances are stable before returning. When UPDATED is set, it will wait for the version target to be reached and any per instance configs to be effective and all instances configs to be effective as well as all instances to be stable before returning.`,
 				<% end -%>
 			},
+			<% unless version == "ga" -%>
+			"stateful_internal_ip": {
+				Type:        schema.TypeSet,
```

Comment:
I need to check for this.

Reply:
To add notes here: If I change typeSet to typeList, when running tests for region_instance, a failure will occur saying it was expecting a Set. Swapping back from List to Set will solve the error.

Reply:
Why is it expected to be a Set?

Reply:
The latest log I have shows:
**2022/11/15 12:47:17 [DEBUG] setting computed for "distribution_policy_zones" from ComputedKeys
2022/11/15 12:47:17 [DEBUG] setting computed for "status" from ComputedKeys
panic: interface conversion: interface {} is []interface {}, not *schema.Set**

Reply:
This looks like a internal type error - since this is a handwritten file, there's a good chance there's something in this file that expects this to be a Set rather than a list. It's probably not going to be solvable with API docs, unfortunately - the API wouldn't know what a schema.Set is, since that's a Hashicorp type.

Does the panic include a stack trace? That might help diagnose where the panic is coming from, which would help in evaluating whether the code can / should be changed.

Reply:
I think I got this resolved, now fixing some importStateVerify errors